### ISSUE TITLE
Support LangVersion on MSBuild projects

### ISSUE THREAD

**Issue by monoman** [OmniSharp/omnisharp-roslyn#200]
Currently my C# 6.0 library project parses incorrectly in Visual Studio Code (gives lots of errors/warnings), because the LangVersion property is not read from the csproj to build correct compiling/parsing options bundles to pass to Roslyn as is done for aspnet5 projects, from my perusal of the code. 

Not sure I can cook a nice PR to add the feature, but if you'd prefer, I can try. :smiley:

**Comment by nosami** [comment]
That would be fantastic if you could do it. You probably want to look at these two files https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp/MSBuild/ProjectFile/ProjectFileInfo.cs and https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp/MSBuild/MSBuildProjectSystem.cs

**Comment by monoman** [comment]
@nosami Working on it! :wrench:

**Comment by nosami** [comment]
@monoman - nice! If you want to test it out, you can set the OMNISHARP environment variable to point to your working copy.

**Comment by monoman** [comment]
Sent a PR #201, untested as my environment in transition from VS 2015 Preview to RC.
Also discovered that OmniSharp (and thus VS Code) defaults to C# 6, but I used VS 2015 Preview to generate my project and it used an outdated version of C# 6. VS 2015 RC gave me the right messages and actions and as I fixed my source, VS Code/OmniSharp also accepted it ok, so this patch is not that important unless you want the reverse, to guarantee some source is compiled with an older version of C# syntax.
